tests/framework/stats/function.py

```diff
@@ -136,7 +136,7 @@ def __call__(self, result: Any) -> Any:
         assert isinstance(result, list)
         assert len(result) > 0
         # pylint: disable=R0123
-        if len(result) is 1:
+        if len(result) == 1:
```

Comment:
We could also get rid of the disable=R0123, right?

Reply:
Good catch. I will remove those too.